.github/workflows/sync-issue-labels-compute.yml

```
@@ -1,207 +1,106 @@
-            fs.writeFileSync('labels.json', JSON.stringify(payload));
-            console.log(`Wrote labels artifact payload for PR #${payload.pr_number}: ${payload.labels.length} labels`);
-
-      - name: Upload labels artifact
-        uses: actions/upload-artifact@bbbca2ddaa5d8feaa63e36b76fdaad77386f024f # v7.0.0
+            const result = { pr_number: prNumber, labels: newLabels };
+            fs.writeFileSync('labels.json', JSON.stringify(result));
+            console.log(`Calculated labels: ${newLabels.join(', ')}`);
+
+      - name: Check for Labels File
+        id: check_file
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:2:25: Double quote to prevent globbing and word splitting

(shellcheck)